docs/docs/quick_start/advanced_features/node_expansion.md

```diff
@@ -4,3 +4,79 @@ sidebar_label: "Node Expansion"
 ---
 
 # Node Expansion
+
+It's one of common requirements for a storage system to expand the capacity by adding a new storage node. HwameiStor provides this feature by the following simple procedure.
+
+## Steps
+
+### 1. Prepare a storage node (e.g. k8s-worker-4, /dev/sdb, SSD disk)
+
+Add a node into the Kubernetes cluster, or select a Kubernetes node. And ensure the node has all the required items which are described in [Prerequisites](../install/prereq.md).
+
+After the new node is already in the Kubernetes cluster, make sure the following HwameiStor pods are already running on this node.
+
+```console
+$ kubectl -n hwameistor get pod -o wide | grep k8s-worker-4
+[root@demo-dev-master-01 ~]# k -n hwameistor get pod -o wide | grep demo-dev-worker-02
+hwameistor-local-disk-manager-c86g5     2/2     Running   0     19h   10.6.182.105      k8s-worker-4   <none>  <none>
+hwameistor-local-storage-s4zbw          2/2     Running   0     19h   192.168.140.82    k8s-worker-4   <none>  <none>
+
+# check for the LSN which is for the metadata of the node's storage
```

Comment:
```suggestion
# check LocalStorageNode to get metadata of the storage node.
```
The original sentence is not easy to understand. Try to rewrite as the above.

Reply:
fixed